src/ResourceManagement/RecoveryServices.Backup/RecoveryServices.Backup.Tests/RecoveryServicesBackupTestBase.cs

```diff
@@ -39,7 +39,7 @@ public class RecoveryServicesBackupTestBase : TestBase, IDisposable
 
         public RecoveryServicesBackupClient BackupClient { get; private set; }
 
-        public RecoveryServicesBackupTestBase(MockContext context)
```

Comment:
We've followed the per-test setup pattern (first model) as suggested in the link.